testsuite/src/it/java/org/graalvm/tests/integration/utils/versions/UsedVersion.java

```diff
@@ -108,8 +108,8 @@ static final class VersionParseHelper {
         //@formatter:off
         private static final Map<Integer, String> GRAAL_MAPPING = Map.of(22, "24.0",
                                                                          23, "24.1",
-                                                                         24, "25.0",
-                                                                         25, "25.1");
+                                                                         24, "24.2",
+                                                                         25, "25.0");
```

Comment:
This is pre-existing. But the mapping was wrong. I wonder which bugs this will surface ;-)